templates/kube-burner.jsonnet

```diff
@@ -0,0 +1,4548 @@
+          orderBy: '5',
+          size: '100',
+        },
+        type: 'terms',
+      },
+      {
+        fake: true,
+        field: 'nodeName.keyword',
+        id: '7',
+        settings: {
+          min_doc_count: '1',
+          order: 'desc',
+          orderBy: '_term',
+          size: '100',
+        },
+        type: 'terms',
+      },
+      {
+        field: 'podName.keyword',
+        id: '2',
+        settings: {
+          min_doc_count: '1',
+          order: 'desc',
+          orderBy: '5',
+          size: '100',
+        },
+        type: 'terms',
+      },
+    ],
+  )
+);
+
+local setup_latency = grafana.graphPanel.new(
+  title='Top 10 Container runtime network setup latency',
+  datasource='$datasource1',
+  legend_alignAsTable=true,
+  legend_max=true,
+  legend_avg=true,
+  legend_values=true,
+  format='µs',
+)
+                      .addTarget(
+  es.target(
+    query='uuid.keyword: $uuid AND metricName.keyword: containerNetworkSetupLatency',
+    timeField='timestamp',
+    alias='{{labels.node.keyword}}',
+    metrics=[
+      {
+        field: 'value',
+        id: '1',
+        meta: {},
+        settings: {},
+        type: 'avg',
+      },
+    ],
+    bucketAggs=[
+      {
+        fake: true,
+        field: 'labels.node.keyword',
+        id: '3',
+        settings: {
+          min_doc_count: '1',
+          order: 'desc',
+          orderBy: '_term',
+          size: '10',
+        },
+        type: 'terms',
+      },
+      {
+        field: 'timestamp',
+        id: '2',
+        settings: {
+          interval: 'auto',
+          min_doc_count: '1',
+          trimEdges: 0,
+        },
+        type: 'date_histogram',
+      },
+    ],
+  )
+);
+
+local scheduling_throughput = grafana.graphPanel.new(
+  title='Scheduling throughput',
+  datasource='$datasource1',
+  legend_alignAsTable=true,
+  legend_max=true,
+  legend_avg=true,
+  legend_values=true,
+  format='reqps',
+)
+                              .addTarget(
+  es.target(
+    query='uuid: $uuid AND metricName.keyword: schedulingThroughput',
+    timeField='timestamp',
+    metrics=[
+      {
+        field: 'value',
+        id: '1',
+        meta: {},
+        settings: {},
+        type: 'avg',
+      },
+    ],
+    bucketAggs=[
+      {
+        field: 'timestamp',
+        id: '2',
+        settings: {
+          interval: 'auto',
+          min_doc_count: '1',
+          trimEdges: 0,
+        },
+        type: 'date_histogram',
+      },
+    ],
+  )
+);
+
+// OVN section
+local ovnkube_master_cpu = grafana.graphPanel.new(
+  title='ovnkube-master CPU usage',
+  datasource='$datasource1',
+  legend_alignAsTable=true,
+  legend_max=true,
+  legend_avg=true,
+  legend_values=true,
+  format='percent',
+)
+                           .addTarget(
+  es.target(
+    query='uuid.keyword: $uuid AND metricName: "containerCPU" AND labels.namespace.keyword: "openshift-ovn-kubernetes"  AND labels.pod.keyword: /ovnkube-master.*/',
```

Comment:
@jaredoconnell nice work, can you update this query to `uuid.keyword: $uuid AND metricName: "containerCPU" AND labels.pod.keyword: /ovnkube-master.*/`